geometry_msgs/TransformStamped data
